components/test_raftstore/src/util.rs

```diff
@@ -595,6 +596,55 @@ pub fn async_read_index_on_peer<T: Simulator>(
     })
 }
 
+pub fn sync_get_snapshot<T: Simulator>(
```

Comment:
```suggestion
pub fn get_snapshot<T: Simulator>(
```
NIT: the `sync_` prefix is redundant